apipie-bindings.gemspec

```diff
@@ -22,17 +22,16 @@ EOF
   s.extra_rdoc_files = `git ls-files -- {doc,config}/* README*`.split("\n")
   s.require_paths    = ["lib"]
 
-  s.add_dependency 'json', '>= 1.2.1'
-  s.add_dependency 'rest-client', '>= 1.6.5', '< 3.0.0'      # lower versions don't allow setting infinite timeouts, higher versions have different api
-  s.add_dependency 'oauth'
+  s.add_dependency 'json', '>= 2.3.0'
```

Comment:
`json` is a default gem since Ruby since quite some time. According to https://stdgems.org/json/ since Ruby 2.3 (also https://stdgems.org/new-in/2.3/) so you can also leave it out these days.
```suggestion
```